src/generators/react-typescript/entity-management/template/EntityManagementEditor.tsx.ejs

```diff
@@ -71,35 +74,31 @@ class <%= editComponentName %>Component extends React.Component<Props & WrappedC
       return <Redirect to={<%= className %>.PATH}/>
     }
 
-    const {getFieldDecorator} = this.props.form;
     const {status} = this.dataInstance;
 
     return (
       <Card className='page-layout-narrow'>
         <Form onSubmit={this.handleSubmit}
               layout='vertical'>
-          <% editAttributes.forEach(attr => { %>
-          <Form.Item label={<Msg entityName={<%= entity.className %>.NAME} propertyName='<%= attr.name %>'/>}
-                    key='<%= attr.name %>'
-                    style={{marginBottom: '12px'}}>{
-
-            getFieldDecorator('<%= attr.name %>', {
-                <% if (attr.mandatory) { %>
-                  rules:[{required: true}],
-                <% } %>
-                <% if (attr.type && attr.type.fqn === 'java.lang.Boolean') { %>
-                  valuePropName:"checked"
-                <% } %>}
-            )(
-                <FormField entityName={<%= entity.className %>.NAME}
-                           propertyName='<%= attr.name %>'
-                        <% if (Object.keys(editRelations).includes(attr.name)) { %>
-                           optionsContainer={this.<%= attr.name %>sDc}
-                        <% } %>
-                />
-            )}
-          </Form.Item>
-          <% }) %>
+
+        <% editAttributes.forEach(attr => { %>
+          <FormItem entityName={<%= entity.className %>.NAME}
+                    propertyName="<%= attr.name %>"
+                    form={ this.props.form }
+
+            <% if (attr.mandatory) { %>
+                    rules={ [{required: true}] }
+            <% } %>
+
+            <% if (attr.type && attr.type.fqn === 'java.lang.Boolean') { %>
+                    valuePropName="checked"
+            <% } %>
```

Comment:
I think it would be beneficial to allow client developer to customize other parameters inside `getFieldDecorator`'s `options` argument. So perhaps we could pass `options` here instead. Similarly, we may want to pass `getFieldDecorator`'s `id` here as an optional prop.